editor/editor_settings.cpp

```diff
@@ -578,6 +578,13 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("text_editor/behavior/navigation/drag_and_drop_selection", true);
 	_initial_set("text_editor/behavior/navigation/stay_in_script_editor_on_node_selected", true);
 
+	// Behavior: Input
+#ifdef LINUXBSD_ENABLED
+	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "text_editor/behavior/input/middle_mouse_button_paste", true, "")
+#else
+	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "text_editor/behavior/input/middle_mouse_button_paste", false, "")
+#endif
```

Comment:
The value shouldn't change depending on the platform, as this will cause unnecessary differences in the output of `--doctool`.

There's no reason to have the setting as false on non-Linux, since it doesn't do anything anyway. So it can be true by default.